### DISCUSSION TITLE
Problem with symbols units

### DISCUSSION THREAD

**Discussion by fjfilipe** [joaomlourenco/novathesis#482] [Q&A]
Hey! I was trying to insert units in my symbols list by doing this:

%% Additional packages area %%

\usepackage{siunitx}
\setglossarystyle{altlong4col}
%\setlength{\glsdescwidth}{0.85\linewidth} %Não obrigatório – variar a largura da descrição

%%% Symbols file %%%

\glsxtrnewsymbol[ description={Massa}, symbol={ [\si{kg}] } ] {m} {\ensuremath{m}}

The error message is the following:
<img width="658" alt="Captura de ecrã 2024-02-01, às 18 47 01" src="https://github.com/joaomlourenco/novathesis/assets/158509453/9aa6906d-99b7-4bed-b99d-dacb7a29fae1">

Can someone please help me solving this?

**Comment by joaomlourenco** [comment]
Comment the line `\setglossarystyle{altlong4col}` and it will probably work. Let me know...

**Reply by fjfilipe** [reply]
Thanks for your help! Unfortunately that doesn't fix my problem. If I comment that line, the error doesn't show up but the units don't appear as well... Maybe im writing it in the worn file. I tried putting it in 5_packages.tex and also glossary.tex, glossary-xltabular.sty and symbols.tex...
